lib/node_modules/@stdlib/array/complex64/README.md

```diff
@@ -1607,6 +1607,46 @@ var count = context.count;
 // returns 2
 ```
 
+<a name="method-subarray"></a>
+
+#### Complex64Array.prototype.subarray( \[begin\[, end]] )
+
+Creates a new typed array view over the same underlying [`ArrayBuffer`][@stdlib/array/buffer] and with the same underlying data type as the host array.
+
+```javascript
+var arr = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0 ] );
+
+var subarr = arr.subarray();
+// returns <Complex64Array>
+
+var len = subarr.length;
+// returns 2
```

Comment:
@Jaysukh-409 We probably want to be a bit more illustrative here in these examples. Just showing the length doesn't really allow readers to understand what is going on.

Instead, perhaps use the `get` method to retrieve the first and last elements in the subarray. I also suggest making the original array a bit longer (e.g., 4 elements) and be consistent across all examples.